src/EditorFeatures/CSharpTest/Diagnostics/HideBase/HideBaseTests.cs

```diff
@@ -93,5 +93,29 @@ class App : Application
     public new int Test;
 }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddNew)]
```

Comment:
You can add a new commit to your branch, and when you push your updated branch to your fork it will automatically update this pull request.